logfire/_internal/main.py

```diff
@@ -1212,8 +1208,7 @@ def instrument_httpx(
         Args:
             client: The `httpx.Client` or `httpx.AsyncClient` instance to instrument.
                 If `None`, the default, all clients will be instrumented.
-            capture_request_headers: Set to `True` to capture all request headers.
-            capture_response_headers: Set to `True` to capture all response headers.
+            capture_headers: Set to `True` to capture all HTTP headers.
```

Comment:
Let's link to the integrations/guide page from the main body of this docstring so that (among other things) people know how to capture specific headers.